apps/aens/test/aens_utils_test.erl

```diff
@@ -13,15 +13,13 @@ to_ascii_test() ->
     ?assertEqual({error, no_registrar},
                  aens_utils:to_ascii(<<"abcdefg">>)),
     ?assertEqual({error, multiple_namespaces},
-                 aens_utils:to_ascii(<<"abcd.efgh.test">>)),
-    ?assertEqual({error, registrar_unknown},
-                 aens_utils:to_ascii(<<"abcd.aettt">>)),
+                 aens_utils:to_ascii(<<"abcd.efgh.aet">>)),
     LongName = list_to_binary(string:lowercase(base58:binary_to_base58(crypto:strong_rand_bytes(1000)))),
     ?assertEqual({error, no_label_in_registrar},
-                 aens_utils:to_ascii(<<".test">>)),
+                 aens_utils:to_ascii(<<".aet">>)),
```

Comment:
same here... before Lima, this should be failing differently then after Lima, but that seems not to be tested